### DISCUSSION TITLE
How can I extend Detectron 2 that detects COCO classes to detect 20 more new object classes?

### DISCUSSION THREAD

**Discussion by monacv** [facebookresearch/detectron2#2945] [Q&A]
Could you please kindly guide me on how I can extend Detectron 2 that already detects COCO classes to detect 20 more new object classes on top of 80 COCO classes?

How about `[PointRed](https://github.com/facebookresearch/detectron2/tree/master/projects/PointRend)` instance segmentation? How can I expand it such that it can create an instance segmentation mask for new object classes (20) besides the current 80 COCO classes?

**Comment by danielcamara** [comment]
Hi, 
Did you succeed in doing it?

**Comment by GitHub-Crow** [comment]
you can change config file like `XXX.yaml`, parameter `NUM_CLASSES` is all your attention.
